src/main/java/com/dataguise/saas/dto/ColumnViewUIFormatterDTO.java

```diff
@@ -0,0 +1,176 @@
+package com.dataguise.saas.dto;
+
+import com.google.gson.Gson;
+
+public class ColumnViewUIFormatterDTO {
+
+    String columnName;
+    String dataType;
+    Integer regexId;
+    String regexGroup;
+    String regexLabel;
+    Boolean isSensitive;
+    Boolean isProtected;
+    Boolean isUnprocessed;
+    Boolean isUnscanned;
+    Boolean isColumnHeaderMatched;
+    Boolean isCleaned;
+    Boolean isSafe;
+    Long hitCount;
+    Long mismatchCount;
+    Long nullCount;
+    Double expressionGroupConfidence;
+
+
+
+    String constraintType;
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocVariableCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/dto/ColumnViewUIFormatterDTO.java L26
    
    Missing a Javadoc comment.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1022250368>*